docs/protos/typescript.md

```diff
@@ -33,81 +36,280 @@ There are a few inputs that the SDK needs in order to be properly configured.
 
 ### API Url
 
-If using Neosync Cloud, the backend api url is: `https://neosync-api.svcs.neosync.dev`
+If you are using Neosync Cloud, the backend api url is: `https://neosync-api.svcs.neosync.dev`
 
 The standard localhost url is: `http://localhost:8080`
 
 ### Account ID
 
 The account ID is necessary for some requests that do not have an obvious identifier like retrieving a list of jobs, or a list of connections.
-This can be found by going into the app on the `/:accountName/settings` page and found in the header.
+You can be find your account ID by going into the Neosync app on the `/:accountName/settings` page and found in the header.
 
 ### API Key
 
-An access token (api key, or user jwt) must be used to access authenticated Neosync environments.
+An access token (API key, or user JWT) must be used to access authenticated Neosync environments.
 For an API Key, this can be created at `/:accountName/settings/api-keys`.
 
-## Usage
-
 For a prime example of how to us this SDK, view the [withNeosyncContext](https://github.com/nucleuscloud/neosync/blob/main/frontend/apps/web/api-only/neosync-context.ts#L23) method in the Neosync app's BFF layer.
 
-### Note on Transports
```

Comment:
This is actually a really important section and will vary based on the environment that the user is invoking our SDK in, which is why it has its own section here and imo should not be removed.

The examples in these docs assume node (the examples show importing from the connect-node lib`, but for example, our frontend uses the connect-web since it's called from the client.

Reply:
ah i see good catch